drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java

```diff
@@ -184,14 +184,17 @@ public void testOperatorDropdownWhenOperatorCanBeSet() {
         page.operatorDropdown(widget -> {
             assertTrue(widget instanceof CEPOperatorsDropdown);
 
-            final CEPOperatorsDropdown operatorsDropdown = (CEPOperatorsDropdown) widget;
+            final CEPOperatorsDropdown dropdown = (CEPOperatorsDropdown) widget;
+
+            verify(dropdown.getBox()).setSelectedIndex(0);
```

Comment:
This is kind of suspicious for me. The `(no operator)` option is inserted into dropdown to position 1 - the last `verify` of this test method so I would expect this:
``` java
verify(dropdown.getBox()).setSelectedIndex(1);
```
instead of
``` java
verify(dropdown.getBox()).setSelectedIndex(0);
```

Am I missing something?